packages/terra-site/src/examples/tabs/TabsComponentNavigation.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+import List from 'terra-list';
+import styles from '../../site.scss';
+
+const propTypes = {
+  onClick: PropTypes.func,
+};
+
+const TabsComponentNavigation = ({ onClick }) =>
+  <List className={[styles['site-component-nav'], styles['sub-nav']]}>
+    <List.Item content={<Link onClick={onClick} to="/site/tabs/modular-centered">Modular - Centered</Link>} />
+    <List.Item content={<Link onClick={onClick} to="/site/tabs/modular-left-aligned">Modular - Left Aligned</Link>} />
+    <List.Item content={<Link onClick={onClick} to="/site/tabs/structural">Structural</Link>} />
```

Comment:
When selecting Tabs in these examples the Tabs will sometimes resize and adjust and it is visually noticeable. Is there a way we can do selection highlighting without causing other Tabs to adjust?

Reply:
I was going to sync with @neilpfeiffer on this. This is caused because the text is bolded on selection in the legacy theme, changing the width of the content. I think we'll need to remove that style.